### PR TITLE
Create hooks for users view

### PR DESCRIPTION
**Pull request by whiteadam2**

### CODE REVIEW

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    };
+                });
+            },
+        }
+    );
+
+    return { users, isUsersLoading };
+}
+
+export function useFetchContacts() {
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:32]: move hooks to different files: useFetchUsers and useFetchContact

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const { api } = useContext<IAppContext>(AppContext);
+
+    const [paginationCnfg, setPaginationCnfg] = useState({ page: 0, size: 5 });
+
+    const { data: contacts, isLoading: isContactsLoading }: any = useQuery(
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:37]: add error: isError

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        }
+    );
+
+    return { users, isUsersLoading };
+}
+
+export function useFetchContacts() {
+    const { api } = useContext<IAppContext>(AppContext);
+
+    const [paginationCnfg, setPaginationCnfg] = useState({ page: 0, size: 5 });
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:35]: use different values for page and size
[page, setPage] = useState(0);
[size, setSize] = useState(5);

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    const { data: users, isLoading: isUsersLoading }: any = useQuery(
+        ['users', paginationCnfg],
+        () => api?.users.readList(paginationCnfg),
+        {
+            keepPreviousData: true,
+            select: (users: any) => {
+                return users.data.map((user: any) => {
+                    return {
+                        ...user,
+                        username: getName(user.userName, user.firstName, user.secondName),
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:20]: If you want to use this utils method pass only one argument user: getUserName(user)

File: apps/monitor-client/lib/features/admin-page/utils/getName.tsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export function getName(...params: any[]): string {
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/admin-page/utils/getName.tsx:1]: function getUserName(user) {
   return `${user.firstName} ${user.secondName}`
}

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+import { useQuery } from 'react-query';
+import { AppContext, IAppContext } from '../../../context';
+
+export function useFetchUsers() {
+    const { api } = useContext<IAppContext>(AppContext);
+
+    const [paginationCnfg, setPaginationCnfg] = useState({ page: 0, size: 5 });
+
+    const { data: users, isLoading: isUsersLoading }: any = useQuery(
+        ['users', paginationCnfg],
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:12]: ['users', paginationCnfg] ==> 'users'

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    return { users, isUsersLoading };
+}
+
+export function useFetchContacts() {
+    const { api } = useContext<IAppContext>(AppContext);
+
+    const [paginationCnfg, setPaginationCnfg] = useState({ page: 0, size: 5 });
+
+    const { data: contacts, isLoading: isContactsLoading }: any = useQuery(
+        ['contacts', paginationCnfg],
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:38]: ['contacts', paginationCnfg] ==> 'contacts'

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx (hunk 1 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:32]: done

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx (hunk 2 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:37]: done

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx (hunk 3 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:35]: done

File: apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx (hunk 4 above)
Comment (whiteadam2) [reply apps/monitor-client/lib/features/admin-page/hooks/useFetchData.tsx:20]: done

**Review by yzhbankov — APPROVED** [review]

**Pull request merged**